src/token/token.service.ts

```diff
@@ -129,11 +129,25 @@ export class TokenService {
     }
 
     if (user.clientIp !== clientIp) {
-      throw new EaseyException(
-        new Error('Request coming from invalid IP address'),
-        HttpStatus.BAD_REQUEST,
-        { userId: user.userId, clientIp: clientIp, storedIp: user.clientIp },
-      );
+      // CHANGED: Log IP change instead of throwing exception
+      this.logger.warn('IP address change detected - audit only', {
+        event: 'IP_CHANGE_DETECTED',
+        userId: user.userId,
+        sessionId: user.sessionId,
+        previousIp: user.clientIp,
+        currentIp: clientIp,
+        timestamp: new Date().toISOString()
+      });
```

Comment:
This should use the `auditLog` message, which formats the log message so the object can be parsed in Cloud.gov. Also, the `warn` log level is currently disabled in upper environments, so this wouldn't be printed.
```suggestion
      this.logger.auditLog({
        event: 'IP_CHANGE_DETECTED',
        userId: user.userId,
        sessionId: user.sessionId,
        previousIp: user.clientIp,
        currentIp: clientIp,
        timestamp: new Date().toISOString()
      });
```
@maheese Will it help reporting if the same structure is used as other audit logs, or does that not really matter? That structure would be:
```typescript
{
  eventContext: "TokenService",
  eventName: "validateClientIp",
  eventOutcome: "IP CHANGE DETECTED",
  eventSource: clientIp,
  userId: user.userId,
  moreInfo: {
    sessionId: user.sessionId,
    previousIp: user.clientIp,
  },
}
```

Reply:
done.